README.md

```diff
@@ -1,22 +1,33 @@
-# Cellar Rebalancer
+# Steward
 
-The Cellar Rebalancer is an appliaction intended for developers and validators on Sommelier network.
+Steward is an application intended for developers and validators on Sommelier network.
 
 It can run in single signer mode or as a voter in the Cosmos sommelier protocol.
 
-It integrates the full functionality of gorc for operating as an orchestator and relayer of gravity bridge messages between Ethereum and Cosmos chains.
+It integrates the full functionality of [Gorc](https://github.com/PeggyJV/gravity-bridge/tree/main/orchestrator/gorc) for operating as an orchestrator and relayer of [Gravity bridge](https://github.com/PeggyJV/gravity-bridge) messages between the Ethereum and Cosmos chains.
 
-## TLS
+Steward is built with the [Abscissa](https://github.com/iqlusioninc/abscissa) app micro-framework.
 
-To provide data to Steward, an encrypted and authenticated gRPC connection must be established. The client certificate authority used by the initial Data Provider is included in `tls/`. This is the only client root of trust accepted by default by Steward right now as we are only accepting client certs from one Data Provider, VolumeFi.
+## Steward use case
+Steward is responsible for running the Orchestrator, which handles relaying Cosmos transactions to Ethereum, and co-processing Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage [Cellars](steward/src/cellars) on Ethereum. To to this, Steward provides a gRPC server to accept recommendations from data providers. 
```

Comment:
I think you meant "To do" rather than "To to". The gRPC server really doesn't have anything to do with the orchestrator though, they're only sharing the steward binary for convenience. Maybe move this last bit to the beginning of the next paragraph to explain the data provider functionality.

Reply:
Done!